src/components/Common/Typography/Link.tsx

```diff
@@ -14,15 +18,19 @@ function isLocalLink(href?: string | null) {
   )
 }
 
+const TARGET_BLANK = '_blank'
+
 export default function Link({ target, rel, href, onClick, className, ...props }: Props) {
   const isLocal = isLocalLink(href)
-  const linkTarget = !target && !isLocal ? '_blank' : target || undefined
+  const linkTarget = !target && !isLocal ? TARGET_BLANK : target || undefined
   const linkRel = !isLocal ? classNames(rel, 'noopener', 'noreferrer') : rel
   const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
     if (onClick) {
       onClick(e)
     }
-    if (isLocal && href) {
+
```

Comment:
And for 10k more `onClick && onClick(e)` instead of the if

Reply:
I prefer the explicit if in this case